jujugui/static/gui/src/app/components/inspector/test-inspector.js

```diff
@@ -44,4 +49,25 @@ describe('Inspector', function() {
           <div>child</div>
         </div>);
   });
+
+  it('calls the changeState callable when on header click', function() {
+    var service = {
+      get: function() {
+        return {name: 'demo'};
+      }};
+    var changeStub = sinon.stub();
+    var component = testUtils.renderIntoDocument(
```

Comment:
If you use the shallow renderer you can test the callback using the same method I did here: https://github.com/juju/juju-gui/pull/828/files#diff-67387ce43fb830bca9478ec54bff89d4R194